folium/utilities.py

```diff
@@ -345,3 +345,42 @@ def png_pack(png_tag, data):
         png_pack(b'IDAT', zlib.compress(raw_data, 9)),
         png_pack(b'IEND', b'')])
  
+def geodetic_to_mercator(geodetic):
+    """This function takes an 2D array in geodetic coordinates (ie: lat x
+    lon unprojected) and converts it to web mercator.  This is needed
+    to correctly overlay an image on a leaflet map, which uses web
+    mercator by default.
+
+    This code works with arrays that match the relative proportions of
+    latitude and longitude of the earth, meaning that they have twice
+    as many points in longitude as latitude (i.e.: (180, 360) for 1
+    degree resolution).
+
+    The code for this is from:
+    http://stackoverflow.com/questions/25058880/convert-to-web-mercator-with-numpy
+
+    Parameters
+    ----------
+    geodetic: 2D numpy array
+        Latitude x Longitude 2D array
+
```

Comment:
Missing a `Returns` here, right?  Although, this function only takes-1 and returns-1, so we could probably remove all this.  It is covered in the first line of the docstring.